docs/src/pages/getting-started/installation/dependencies.react.mdx

```diff
@@ -51,3 +51,44 @@ export default function App(props) {
 ```
 </TabItem>
 </Tabs>
+
+If you're planning on using the [Authenticator](/components/authenticator), also wrap your application in `<Authenticator.Provider/>`.
```

Comment:
The `Authenticator` is wrapped with a `Provider` internally, so there is no need to wrap an app that uses the `Authenticator` within an `Authenticator.Provider` **unless** the user is attempting to utilize `useAuthenticator` outside the scope of the `Authenticator`, e.g.

```tsx
const ContrivedExample = () => {
  const { user } = useAuthenticator(); // will throw if not wrapped in Authenticator.Provider
  return user.userName ? <p>Hi {user.userName}</p> : <Authenticator />;
}

const App = () => {
  return (
    <AmplifyProvider>
      <Authenticator.Provider>
        <ContrivedExample />
      </Authenticator.Provider>
    </AmplifyProvider>
  )
}
```

Of course the above example is better handled by just moving `ContrivedExample` inside the `Authenticator`:

```tsx
const ContrivedExample = () => {
  const { user } = useAuthenticator(); // works, as it's wrapped in an Authenticator
  return <p>Hi {user.userName}</p>;
}

const App = () => {
  return (
    <AmplifyProvider>
      <Authenticator>
        <ContrivedExample />
      </Authenticator>
    </AmplifyProvider>
  )
}
```

Reply:
Correct. I hinted at that in the note at the bottom. That's why I think we should include it for everyone using Authenticator, for that scenario, so it's not confusing.

Reply:
Updated

Reply:
But isn't that directing our users to include superfluous elements in their rendering tree?

Reply:
OK I moved this around to two different sections, based on our offline discussion